torch/csrc/jit/passes/frozen_graph_optimizations.cpp

```diff
@@ -14,6 +15,7 @@ void OptimizeFrozenGraph(std::shared_ptr<Graph>& graph) {
     FoldFrozenConvBatchnorm(graph);
     FoldFrozenConvAddOrSub(graph);
     FoldFrozenConvMulOrDiv(graph);
+    removeDropout(graph);
```

Comment:
Nit: the other passes in this are run multiple times to catch different patterns of Conv-BatchNorm-Add / Conv-Add-Mul / Conv-Mul-Add. There isn't any reason to run dropout removal twice; they should be removed after the first call.